packages/sanity/src/core/releases/tool/components/releaseCTAButtons/ReleaseRevertButton/usePostPublishTransactions.ts

```diff
@@ -0,0 +1,37 @@
+import {useMemo} from 'react'
+import {useObservable} from 'react-rx'
+import {catchError, filter, from, map, of} from 'rxjs'
+
+import {useClient} from '../../../../../hooks/useClient'
+import {getTransactionsLogs} from '../../../../../store/translog/getTransactionLogs'
+import {API_VERSION} from '../../../../../tasks/constants'
+import {type DocumentInRelease} from '../../../detail/useBundleDocuments'
+
+export const usePostPublishTransactions = (documents: DocumentInRelease[]) => {
+  const client = useClient({apiVersion: API_VERSION})
+  const transactionId = documents[0]?.document._rev
+
+  const memoHasPostPublishTransactions = useMemo(() => {
+    if (!documents.length) return of(false)
+
+    return from(
+      getTransactionsLogs(
+        client,
+        documents.map(({document}) => document._id),
+        {
+          fromTransaction: transactionId,
+          // publish transaction + at least one post publish transaction
+          limit: 2,
+        },
+      ),
+    ).pipe(
+      filter(Boolean),
+      // the transaction of published is also returned
+      // so post publish transactions will result in more than 1 transaction
+      map((transactions) => transactions.length > 1),
+      catchError(() => of(null)),
```

Comment:
Are we sure we want to silently ignore any error here? Could you add a short commend telling the next reader why it's safe/ok if so?

To me it seems like any failure here could lead us falsely assume there is no transactions, which might leave the end-user with a wrong impression